### ISSUE TITLE
[BUG]: Snipping Tool still can be executable via its keyboard shortcut

### ISSUE THREAD

**Issue by ghost** [undergroundwires/privacy.sexy#343]
### Description
I deleted Snipping Tool via privacy.sexy to use Win+Shift+S shortcut for another application and restarted my pc, but yeah it still executes the Snipping tool it shows rectangle stuff.

### OS
Windows 10 IOT LTSC 21

### Reproduction steps
Just delete Snipping tool in Bloatware section and run win+shift+s does not matter if you reboot or not it still executes itself.

### Scripts
```
@echo off
:: https://privacy.sexy — v0.13.1 — Sun, 14 Apr 2024 02:25:33 GMT
:: Ensure admin privileges
fltmc >nul 2>&1 || (
    echo Administrator privileges are required.
    PowerShell Start -Verb RunAs '%0' 2> nul || (
        echo Right-click on the script and select "Run as administrator".
        pause & exit 1
    )
    exit 0
)
:: Initialize environment
setlocal EnableExtensions DisableDelayedExpansion


:: ----------------------------------------------------------
:: ----------------Remove "Snipping Tool" app----------------
:: ----------------------------------------------------------
echo --- Remove "Snipping Tool" app
:: Uninstall 'Microsoft.ScreenSketch' Microsoft Store app.
PowerShell -ExecutionPolicy Unrestricted -Command "Get-AppxPackage 'Microsoft.ScreenSketch' | Remove-AppxPackage"
:: Mark 'Microsoft.ScreenSketch' as deprovisioned to block reinstall during Windows updates.
reg add "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\Appx\AppxAllUserStore\Deprovisioned\Microsoft.ScreenSketch_8wekyb3d8bbwe" /f
:: ----------------------------------------------------------


:: Pause the script to view the final state
pause
:: Restore previous environment settings
endlocal
:: Exit the script successfully
exit /b 0
```

**Comment by undergroundwires** [comment]
Hi, thanks for the report.

It works on older  versions but snipping tool is not provided as separate store app but as part of `MicrosoftWindows.Client.Core` (Windows Feature Experience Pack) on modern versions of Windows.

I will not add functionality delete this package. It includes a lot of other useful things and would have big impact to delete it. I think we can add group policy to achieve this.

But we can disable it:

```batchfile
reg add "HKCU\Control Panel\Keyboard" /v "PrintScreenKeyForSnippingEnabled" /t "REG_DWORD" /d "0" /f
reg add "HKLM\SOFTWARE\Policies\Microsoft\TabletPC" /v "DisableSnippingTool" /t "REG_DWORD" /d "1" /f
```

Let me know if it works.

We can introduce this as new script: **Disable Snipping Tool** as this tool is historically known to have vulnerabilities [1] and disabling would reduce your attack surface.

[1]: https://msrc.microsoft.com/update-guide/vulnerability/CVE-2023-28303

**Comment by undergroundwires** [comment]
Restart your computer and please get back if it worked.

It did not work for me until reboot but works fine afterwards.

I see that this is what control panel option (*Settings* > *Accessibility* > *Keyboard* > *Use the Print screen key to open screen capture*) does too. It sets `HKCU\Control Panel\Keyboard!PrintScreenKeyForSnippingEnabled` to `0`  (tested on Windows 11 23H2) and does something else to not require reboot.

I see that users of other screenshot software posted related issues flameshot-org/flameshot#1551, greenshot/greenshot#482.

**Comment by undergroundwires** [comment]
Thank you for coming back. Try this:

 ```batchfile
 reg add "HKCU\Software\Microsoft\Windows\CurrentVersion\Explorer\Advanced" /v "DisabledHotkeys" /t "REG_SZ" /d "S" /f
 ```
 
 After running this, try directly if it works, if it doesn't work, try restarting explorer:
 
 ```batchfile
 taskkill /f /im explorer.exe & start explorer
 ```
 
 If it doesn't work try system restart.

Check if it works, and feel free to let me know if restarting explorer or computer was needed.

**Comment by undergroundwires** [comment]
I will update the code in a patch release, did you need to restart the computer or was restarting explorer.exe enough?

**Comment by undergroundwires** [comment]
🎉 Released since [0.13.3](https://github.com/undergroundwires/privacy.sexy/releases/edit/0.13.3).
